crates/router/src/services/api.rs

```diff
@@ -240,7 +240,14 @@ async fn send_request(
             .await
         }
 
-        Method::Put => client.put(url).add_headers(headers).send().await,
+        Method::Put => {
+            client
+                .put(url)
+                .add_headers(headers)
+                .body(request.payload.expose_option().unwrap_or_default())
```

Comment:
Why we need to unwrap with the default payload? any specific reason. If we can able to build the payload then it should be a error.

Reply:
If payload is not being processed, we can use default for now, but in future if payload processing comes we can't do `unwrap_or_default()`